examples/analog_hamiltonian_simulation/00_Introduction_of_Analog_Hamiltonian_Simulation_with_Rydberg_Atoms.ipynb

```diff
@@ -21,9 +21,9 @@
    "id": "b327221a-aee8-4621-b657-ea85c750a09f",
```

Comment:
<p>This is a great section. Awesome details. I'm thinking maybe we can use the 3-atom layout AND use the local shift to detune one of the atoms far from resonance. This will effectively remove it from the equation, and your analytic result will hold.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>added</p>